packages/ui/package.json

```diff
@@ -125,6 +125,8 @@
     "antd": "^4.24.3",
     "classnames": "^2.3.2",
     "react-markdown": "^6",
-    "remark-gfm": "^1"
+    "remark-directive": "^1",
+    "remark-gfm": "^1",
```

Comment:
These packages have newer versions. Why we are using two-year-old ones?

Reply:
we are using react-markdown package version 6, we can't update it at this time because if we build our apps the "document is not defined error" and react-markdown version 6 is only compatible with older versions of these packages